include/boost/beast/experimental/core/url_parser.hpp

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (c) 2018 jackarain (jack dot wgm at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/beast
+//
+
+#ifndef BOOST_BEAST_EXPERIMENTAL_CORE_URL_PARSER_HPP
```

Comment:
I don't bother mentioning EXPERIMENTAL in the macro:
```
#ifndef BOOST_BEAST_CORE_URL_PARSER_HPP
```
Code is structured "as-if" it was public, except that it is place in the experimental/ directory. So if we never decide to publish it, then all that is necessary is to delete "experimental/" from all include paths and apply suitable file renaming.

Reply:
Oh, that makes complete sense, I like it. I'll make the edits.